src/main/resources/application.properties

```diff
@@ -1,2 +1,13 @@
 server.port=5000
-spring.data.mongodb.uri=mongodb+srv://<username>:<password>@cluster0-nomfh.mongodb.net/InternHome
+#spring.data.mongodb.uri=mongodb+srv://<username>:<password>@cluster0-nomfh.mongodb.net/InternHome
+#user: db_user
+#pass: vloUr7AakeTEdXDj
+
+spring.data.mongodb.uri=mongodb+srv://db_user:vloUr7AakeTEdXDj@cluster0-nomfh.mongodb.net/InternHome
+#spring.data.mongodb.uri=mongodb+srv://Ale_dbuser:Alejandro@cluster0-sas3p.mongodb.net/test?retryWrites=true&w=majority/intern
+
+#spring.data.mongodb.uri=mongodb+srv://Ale_dbuser:intern@cluster0-sas3p.mongodb.net/intern
+#spring.data.mongodb.uri=mongodb+srv://Ale_dbuser:intern@cluster0-sas3p.mongodb.net/test?retryWrites=true&w=majority/intern
+spring.data.mongodb.database = InternHome
+
+#5e6290031c9d44000033734d~
```

Comment:
Please delete all unnecessary properties and replace the password to < password >.